README.md

```diff
@@ -1,93 +1,113 @@
 Introduction
-============
+=============
 
-This app provides an "exam" app that lets users take an online
-programming quiz.  Currently only Python and simple Bash scripts can be
-tested.  At FOSSEE, Nishanth had implemented a nice django based app to
-test for multiple-choice questions.  However, I was inspired by a
+This project provides an "exam" app that lets users take an online
+programming quiz. 
+
+#### Inspiration
+
+At FOSSEE, Nishanth had implemented a nice django based app to
+test for multiple-choice questions. I was inspired by a
 programming contest that I saw at PyCon APAC 2011.  Chris Boesch, who
-administered the contest, used a nice web application that he had built
-on top of GAE that basically checked your Python code, live.  This made
-it fun and interesting.  Their application can be seen at
-http://singpath.com
+administered the contest, used a nice web application 
+[Singpath](http://singpath.com) that he had built on top of GAE that 
+basically checked your Python code, live. This made it fun and interesting.
 
 I wanted an implementation that was not tied to GAE and decided to write
 one myself and the result is the "exam" app.  The idea being that I can
 use this to test students programming skills and not have to worry about
 grading their answers myself and I can do so on my machines.
 
-You can define fairly complicated programming problems and have users
-solve the problem and the solution is checked immediately. The system
-supports pretty much arbitrary Python and uses "test cases" to test the
-implementations of the students.  It also supports simple bash scripts
--- see the sample questions in "docs/".  In addition it supports simple
-multiple choice questions.  Since it runs on your Python, you could
-technically test any Python based library.  It is distributed under the
-BSD license.
+#### Features
+
+ * Define fairly complicated programming problems and have users
+ solve the problem. 
+ * Immediate verification of code solution. 
+ * Supports pretty much arbitrary coding questions in Python, C, C++ and 
+ simple Bash and uses "test cases" to test the implementations of the students.
+ * Supports simple multiple choice questions and File uploads.
+ * Since it runs on your Python, you could technically test any Python based library.
+ * Scales to over 500+ simultaneous users.
+ * Distributed under the BSD license.
+
+Quick Start for Users
+======================
+
+#### Pre-Requisite
+
+1. [Install pip](https://pip.pypa.io/en/latest/installing.html) 
```

Comment:
Make sure you have pip available and usable might be a better usage.

So what does someone on a Linux machine without root access do?  There should be a section that you can link to for the benefit of those users.